ide/gwt-logger/pom.xml

```diff
@@ -42,29 +42,76 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
-    <build>
-        <resources>
-            <resource>
-                <directory>src/main/java</directory>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-            </resource>
-            <resource>
-                <directory>${project.build.outputDirectory}/org/eclipse/che/ide/logger/slf4j/emulation</directory>
-            </resource>
-        </resources>
-        <plugins>
-            <plugin>
+	<build>
+		<plugins>
```

Comment:
@tsmaeder you've introduced "tab" characters while the file is formatted with spaces.

what you can do is:
```
$  cd ide/gwt-logger
$ mvn -o com.google.code.sortpom:maven-sortpom-plugin:sort
```